kubernetes-manifests/adservice.yaml

```diff
@@ -35,10 +35,10 @@ spec:
         env:
         - name: PORT
           value: "9555"
-        # - name: DISABLE_STATS
-        #   value: "1"
-        # - name: DISABLE_TRACING
-        #   value: "1"
+        - name: DISABLE_STATS
+          value: "1"
+        - name: DISABLE_TRACING
+          value: "1"
```

Comment:
Question: Do you know if disabling Cloud Tracing, etc. by default would require adding an extra step in some of the existing tutorials?
* For instance, could it break this [Exporting to BigQuery tutorial](https://cloud.google.com/trace/docs/trace-export-bigquery#hipstershop-query)? I can't really tell — because this tutorial is a bit vague.

Reply:
It's interesting because this tutorial doesn't have the user deploy the app. It just assumes they already have. 

I will update the readme to add a notice about how to re-enable GCP instrumentation, overriding the default.